.github/workflows/ci.yml

```
@@ -40,6 +40,18 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: 🚫 main PR 소스 브랜치 검증
+        if: github.base_ref == 'main'
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)